system-contract-dapp-playground/__tests__/hedera/erc20-interactions/index.test.ts

```diff
@@ -0,0 +1,71 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { getERC20TokenInformation } from '@/api/hedera/erc20-interactions';
```

Comment:
Interesting.
Q: Would import by relative file path not work and would this require you to compile to get any new changes?

Reply:
Hi Nana thanks for the question. Regards to the first part of the question, importing by relative file paths would work fine but I find using import aliases is cleaner. The import statement above when use relative paths would look sort of like this 
```
import { getERC20TokenInformation } from '../../../src/api/hedera/erc20-interactions';
```

For the second part of the question, if I understand your Q correctly then when new changes added it doesn't require to be recompiled. The import aliases are resolved at compile-time and used by the tsc to generate the correct import path in Javascript. After the code is done compiled, the import aliases are not relevant any more. It's a setting in the tsconfig file which can be found here https://github.com/hashgraph/hedera-smart-contracts/blob/main/system-contract-dapp-playground/tsconfig.json#L23

Reply:
Nice, thanks for the details 😃